utils/types.go

```diff
@@ -0,0 +1,44 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// Package utils contains helper functions and common types used by pmm-managed.
+package utils
+
+import (
+	"github.com/percona-platform/saas/pkg/check"
+
+	"github.com/percona/pmm-managed/models"
+)
+
+// Target contains required info about STT check target.
+type Target struct {
+	AgentID       string
+	ServiceID     string
+	ServiceName   string
+	Labels        map[string]string
+	Dsn           string
```

Comment:
let's use consistent case for acronyms (the same for `Tdp `)
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
fixed